modules/navier_stokes/doc/content/source/fvkernels/PINSFVMomentumFriction.md

```diff
@@ -2,19 +2,67 @@
 
 This kernel adds the friction term to the porous media Navier Stokes momentum
 equations. This kernel must be used with the canonical PINSFV variable set,
-e.g. pressure and superficial velocity.  A variety of models are supported for
-the friction force:
+e.g. pressure and superficial velocity. This kernel supports Darcy and
+Forchheimer friction models:
 
 Darcy drag model
 \begin{equation}
-F_i = - \dfrac{f_i}{\epsilon} \rho v_d
+\epsilon F_i = - f_i \frac{\rho v_{D,i}}{\epsilon}
 \end{equation}
 Forchheimer drag model
 \begin{equation}
-F_i = - \dfrac{f_i}{\epsilon} \rho v_d
+\epsilon F_i = - f_i(v) \frac{\rho v_{D,i}}{\epsilon}
+\end{equation}
+where $F_i$ is the i-th component of the friction force (denoted by $\mathbf{F_f}$ in [!eqref](pinsfv.md#eq:pinsfv_mom)), $f_i$ the friction factor, which may be anisotropic,
+$\epsilon$ the porosity and $\rho$ the fluid density and $v_{D,i}$ the i-th
+component of the fluid
+superficial velocity. We have used a negative sign to match the notation used in
+[!eqref](pinsfv.md#eq:pinsfv_mom) where the friction force is on the
+right-hand-side of the equation. When moved to the left-hand side, which is done
+when setting up a Newton scheme, the term becomes positive which is what is
+shown in the source code itself.
+Though the functional forms above are identical and their
+treatment in this kernel is identical, there is a fundamental conceptual
+difference. Darcy is meant to represent viscous effects and should
+have a linear dependence on the fluid velocity, whereas Forchheimer is meant to
+represent inertial effects and should have a quadratic dependence on velocity. These additional dependencies, as well as other prefactors commonly present in the Darcy and Forchheimer models, are baked into the definition of the friction factors.
+
+## Computation of friction factors and pre-factors
+
+To outline how friction factors for Darcy and Forchheimer may be calculated,
+let's consider a specific example. We'll draw from the Ergun equation, which is
+outlined [here](https://en.wikipedia.org/wiki/Ergun_equation). Let's consider
+the form:
+
+\begin{equation}
+\Delta p = \frac{150\mu L}{d_p^2} \frac{(1-\epsilon)^2}{\epsilon^3} v_{D,i} + \frac{1.75 L \rho}{d_p} \frac{(1-\epsilon)}{\epsilon^3} |\vec{v}| v_{D,i}
+\end{equation}
+
+where $L$ is the bed length, $\mu$ is the fluid dynamic viscosity and $d_p$ is
+representative of the diameter of the pebbles in the pebble bed. We can divide
+the equation through by $L$, multiply the equation through by $\epsilon$, and do
+some term manipulation in order to yield:
+
+\begin{equation}
+0 = -\epsilon \nabla p + \left(-150 \frac{\mu\epsilon}{\rho} \frac{1-\epsilon}{\epsilon d_p}^2 \frac{\rho v_{D,i}}{\epsilon} - 1.75 \frac{1-\epsilon}{\epsilon d_p} |\vec{v}| \frac{\rho v_{D,i}}{\epsilon}\right)
```

Comment:
in case this PR gets merged one day, the \epsilon \nabla p term is missing the division by L

Reply:
yup you're right, I read it wrong as \Delta, sorry!